packages/accounts-controller/CHANGELOG.md

```diff
@@ -7,6 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [17.1.0]
+
+### Added
+
+- Add `listMultichainAccounts` action ([#4426](https://github.com/MetaMask/core/pull/4426))
+
+### Fixed
+
+- Update `handleAccountRemoved` to one update ([#4322](https://github.com/MetaMask/core/pull/4322))
```

Comment:
Hmm, this entry seems to undersell what this PR is doing. I notice that the changelog entry left in this PR reads "Updates handleAccountRemoved to update the selected account to the most recent one, if the account being removed is currently the selected account, all within the same step." That's a lot more informative! One thing to note is that this talks about `handleAccountRemoved`. This is a private method in the API, so consumers won't able to see this. Is there another way to describe the changes to the logic here without talking about implementation details? In addition, it looks like there were changes to `getAccountExpect` and `getSelectedAccount` methods included in this PR which aren't reflected here.

Reply:
The PR is mainly just refactoring the internal logic.